qemu/tests/qemu_guest_agent.py

```diff
@@ -802,29 +797,39 @@ def _action_before_fsfreeze(self, *args):
 
     @error_context.context_aware
     def _action_after_fsfreeze(self, *args):
-        error_context.context("Verfiy FS is frozen in guest.")
-        if not isinstance(args, tuple):
-            return
+        error_context.context("Verfiy FS is frozen in guest.", logging.info)
 
         if not self._open_session_list:
-            raise error.TestError("Could not find any opened session")
+            self.test.error("Could not find any opened session")
         # Use the last opened session to send cmd.
         session = self._open_session_list[-1]
+
         try:
             session.cmd(self.params["gagent_fs_test_cmd"])
         except aexpect.ShellTimeoutError:
-            logging.debug("FS freeze successfully.")
+            logging.info("FS is frozen as expected,can't write in guest.")
         else:
-            raise error.TestFail("FS freeze failed, guest still can"
-                                 " write file")
+            self.test.fail("FS is not frozen,still can write in guest.")
 
     @error_context.context_aware
     def _action_before_fsthaw(self, *args):
         pass
 
     @error_context.context_aware
     def _action_after_fsthaw(self, *args):
-        pass
+        error_context.context("Verify FS is thawed in guest.", logging.info)
+
+        if not self._open_session_list:
+            self.test.error("Could not find any opened session.")
```

Comment:
I think this shouldn't be an error, is there some reason to do so?

Reply:
I modify it with creating a session.